lab3/src/pages/dashboard/calendar/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Main Page</title>
+    <link rel="stylesheet" href="/styles/index.css" />
+  </head>
+  <body>
+    <div class="dashboard">
+      <div class="sidebar" id="sidebar">
+        <img src="/assets/images/logo.png" alt="" class="sidebar__logo" />
+
+        <div class="sidebar-item my-4" id="sidebar-toggle">
```

Comment:
This is clickable, innit?

Reply:
yes clickable but if <div> its work because of js and if <a> its work like a link without js

Reply:
ur tags should be semantically correct. If it's opener for a side menu it could be `checkbox` as it has two states - open/close. or `a` if it redirects somewhere, or `button` if any other behaviour is expected

Reply:
ok

Reply:
changed on a